src/ui/src/templates/reset_interval_modal.html

```diff
@@ -0,0 +1,18 @@
+<div class="modal-header">
+    <h4>Reset the Job Interval?</h4>
```

Comment:
It would be good to provide some additional wording here, since I'm not sure most people will understand what this is asking.  Something like:

**Reset the Job Interval?**
Select "Yes" to update the next run time of this job based on the time of this current run.
Select "No" to keep the job's existing next run time.

That's not fantastic so feel free to workshop it a bit.  The point is that I think we need something to convey what this option is about.

Reply:
Yes, I was waiting for your feedback on this after you'd mentioned the issue originally (but I didn't take notes at that time).

Reply:
Made the update with some minor wordsmithing.

Reply:
You somehow came up with something more wordy than I did :rofl:.

My only objection to what you have now is the use of the phrase "original job", which even gets special emphasis in one instance.  This seems very confusing, since from the user's perspective there is just the job that they're trying to interact with.  The fact that they are technically spawning a new job on the backend and that it's the next run time on that "original job" that gets impacted is all just implementation details, so I think referencing the "original job" is confusing.

I'm happy to save any further wordsmithing until we come back and actually enable that pop up.  Since it won't show up yet anyway we can probably just proceed.

Reply:
No, I'd be interested to know what you think it should say. And then I'll make it say that :-)

Reply:
Further minor wording changes made.